pkg/app/app.go

```diff
@@ -164,9 +165,36 @@ func (a *App) Start() {
 		a.log.Notice("Broker successfully bootstrapped on startup")
 	}
 
+	interval, err := time.ParseDuration(a.config.Broker.RefreshInterval)
+	a.log.Debug("RefreshInterval: %v", interval.String())
+	if err != nil {
+		a.log.Error(err.Error())
+		a.log.Error("Not using a refresh interval")
```

Comment:
You'll get the value in the message right before the error check ```a.log.Debug("RefreshInterval: %v", interval.String())```.  Also, I checked all the error messages returned from PraseDuration and they return the interval.  So even if someone doesn't have debug logging level on, they'll get the value in the error plus our message from the broker.